packages/data-service/src/connect-mongo-client.ts

```diff
@@ -73,6 +74,9 @@ export async function connectMongoClientCompass(
   );
 
   const oidcOptions = prepareOIDCOptions(connectionOptions);
+  // @ts-expect-error Will go away on @types/node update
+  // with proper `AbortSignal` typings
+  oidcOptions.oidc.signal = signal;
```

Comment:
Small thing, fine as is here too, how do you feel about moving this line into `prepareOIDCOptions`. (Then we can update the unit tests for that function).

Reply:
@anemy Is something like 9c9c929b0323c28f580908219c1fe8c383ebb301 what you had in mind here?